### PR TITLE
Isn't that YOUR job?

### PR DESCRIPTION
**Pull request by alixgeorge**
### Blog title
Isn't that YOUR job?

#### Author
- [X] I have checked for grammatical errors
- [X] I have checked for spelling errors
- [X] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

File: _posts/2015-11-06--isn-t-that-your-job.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+author: ""
+categories: ""
+tags: ""
+published: false
+title: "'Isn't that YOUR job?'"
+---
+
+## Product Owners and Scrum Masters - Text book vs. Agile
+
+When you read the theory, there should be a clear division between the roles and responsibilities of PO's and SM's.
```
Comment (dancork) [inline _posts/2015-11-06--isn-t-that-your-job.md]: POs and SMs - though may be better to use the full job title in first para

File: _posts/2015-11-06--isn-t-that-your-job.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+tags: ""
+published: false
+title: "'Isn't that YOUR job?'"
+---
+
+## Product Owners and Scrum Masters - Text book vs. Agile
+
+When you read the theory, there should be a clear division between the roles and responsibilities of PO's and SM's. 
+
+PO's own the product; the vision and strategy, and focus on prioritising the work each Sprint to drive the team forward in delivering that vision.
```
Comment (dancork) [inline _posts/2015-11-06--isn-t-that-your-job.md]: POs

File: _posts/2015-11-06--isn-t-that-your-job.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+SM's are instrumental in balancing the facilitation of that vision (the process) and also protecting the team. They're the team coach and motivator, supporting both the engineering people and the PO.
```
Comment (dancork) [inline _posts/2015-11-06--isn-t-that-your-job.md]: SMs

File: _posts/2015-11-06--isn-t-that-your-job.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+## Product Owners and Scrum Masters - Text book vs. Agile
+
+When you read the theory, there should be a clear division between the roles and responsibilities of PO's and SM's. 
+
+PO's own the product; the vision and strategy, and focus on prioritising the work each Sprint to drive the team forward in delivering that vision.
+
+SM's are instrumental in balancing the facilitation of that vision (the process) and also protecting the team. They're the team coach and motivator, supporting both the engineering people and the PO. 
+
+For some teams, this works. They draw clear lines of responsibility and make sure neither stray over into the other's _'realm'_.
```
Comment (dancork) [inline _posts/2015-11-06--isn-t-that-your-job.md]: No comma `For some teams, this works`

File: _posts/2015-11-06--isn-t-that-your-job.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+When you read the theory, there should be a clear division between the roles and responsibilities of PO's and SM's. 
+
+PO's own the product; the vision and strategy, and focus on prioritising the work each Sprint to drive the team forward in delivering that vision.
+
+SM's are instrumental in balancing the facilitation of that vision (the process) and also protecting the team. They're the team coach and motivator, supporting both the engineering people and the PO. 
+
+For some teams, this works. They draw clear lines of responsibility and make sure neither stray over into the other's _'realm'_. 
+
+When I've had conversations with other PO's at conferences and described a day in the office, I've sometimes been meet with a grimace and a 'Aren't those the sort of questions the SM should be asking?'
```
Comment (dancork) [inline _posts/2015-11-06--isn-t-that-your-job.md]: Maybe use double quotes as using single in Aren't

`"Aren't those the sort of questions the SM should be asking?"`

File: _posts/2015-11-06--isn-t-that-your-job.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+When I've had conversations with other PO's at conferences and described a day in the office, I've sometimes been meet with a grimace and a 'Aren't those the sort of questions the SM should be asking?' 
+
+The perception seems to be that even asking a simple question about the progress of a piece of work means I'm straying into SM territory. That is so **not** agile!
+
+That doesn't work for me. The clue is in the name _'agile'_. We should be flexible and bend to what works best for the team to meet the goals of the business.
+
+For me, I see the PO/SM relationship as a partnership of equals. As a PO I should be empowered to motivate the team and flag up processes I think could be better, just as the SM should feel confident in challenging the priorities and suggesting ideas for the next iteration of a project!
+
+We're more than just our job titles, and we all have various skills and strengths we can (and should) bring to the table if it's of benefit to the wider business and the team.
```
Comment (dancork) [inline _posts/2015-11-06--isn-t-that-your-job.md]: `we can, and should, bring to the table`

File: _posts/2015-11-06--isn-t-that-your-job.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+If a piece of work needed testing to get it over the line and complete the Sprint but it'd take too long for the tester to complete it on their own, both the SM and PO should be jumping in to assist! This is _'agile'_ behaviour and is actively encouraged in our business.
+
+We break down those role divisions when we need to because that's what it means to be a team.
+
+Sure, there are things that remain the firm domain of either role. As a Product Owner, it's my job to develop the product vision and the strategy for how we get there, and Scrum Masters will always own process, but when it comes to people and 'being one team', there should be no dividing lines.
+
+Having this partnership with the team's SM has meant that we've developed a strong foundation to challenge, feedback, support and trust each other. 
+
+This dynamic seems to work really well in the our team, but every team is different and that's the point - find out what works best in your team.
```
Comment (dancork) [inline _posts/2015-11-06--isn-t-that-your-job.md]: Point is great but uses the work team 3 times. Maybe something like...

`This dynamic seems to work really well for us, but every team is different and that's the point - find out what works best in you and your co-workers.`

File: _posts/2015-11-06--isn-t-that-your-job.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+We don't have to stick to text-book and **that** is being agile.
```
Comment (dancork) [inline _posts/2015-11-06--isn-t-that-your-job.md]: `the textbook`

**Comment by alixgeorge** [comment]
I've addressed your comments @dancork

**Comment by alixgeorge** [comment]
All comments now addressed @dancork - I've left the use of the word 'team' twice in the penultimate paragraph because I think this reinforces the point but everything else is corrected.

**Comment by dancork** [comment]
Great article :+1:

**Comment by andrew-c** [comment]
:+1:

**Comment by pmcnr-hx** [comment]
Excellent post @alixgeorge! I'll be commenting on it once it's published.

**Comment by alixgeorge** [comment]
Thanks @pmcnr-hx - it'll be going onto the blog next week. :)

**Comment by petebish** [comment]
Great article @alixgeorge . Addresses an issue that i suspect some people may have not even considered, but one that is very real and for those reading this, should at least get them asking some questions of how they embrace Agile as a collective group of differing disciplines, but all aiming for a common goal.

**Pull request merged**